src/lib/features/export-import-toggles/export-import.e2e.test.ts

```diff
@@ -331,6 +353,15 @@ test('exports features', async () => {
                 ],
             },
         ],
+        links: [
```

Comment:
this is a format modelled after the dependencies handling. Link of objects with feature name and corresponding links. This is easy to handle on the import side